x-pack/plugins/observability/public/pages/slos/components/slo_list_item.tsx

```diff
@@ -202,7 +204,7 @@ export function SloListItem({
                   onClick={handleViewDetails}
                   data-test-subj="sloActionsView"
                 >
-                  {i18n.translate('xpack.observability.slo.slo.item.actions.details', {
```

Comment:
One of the `slo`s seemed redundant to me, let me know if there was a reason for it.

Reply:
The initial thoughts was to attribute the first one to the overall slo feature, and the second one to the current main page component, but we haven't done a good job at it.
So it's ok like that 👍🏻